DocFx.Plugins.Kroki/Renderers/KrokiRenderer.cs

```diff
@@ -28,10 +28,19 @@ public sealed override StringBuffer Render(IMarkdownRenderer renderer, MarkdownC
     {
       var formatter = GetFormatter(_settings.OutputFormat, renderer.Options, DiagramType);
       var client = new KrokiClient(_settings.ServiceUrl);
-      var byteData = client.GetDiagram(new KrokiPayload(token.Code, DiagramType, _settings.OutputFormat)).Result;
+      var code = CodeCleanup(token.Code);
+      var byteData = client.GetDiagram(new KrokiPayload(code, DiagramType, _settings.OutputFormat)).Result;
       return formatter.FormatDiagramData(byteData);
     }
 
+    private static string CodeCleanup(string raw)
+    {
+      // Markdown in source /// comments are preprocessed to escape XML reserved characters.
+      return raw.Replace("&amp;", "&")
```

Comment:
I think `System.Net.WebUtility.DecodeHtml` might be a better solution here but it would require some testing.